src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/copyQuestionModal.tag

```diff
@@ -13,43 +14,15 @@
             </div>
-            <div class="modal-body padding-0">
+            <div class="modal-body">
                 <form class="form" id="copyModalForm" role="form" method="post"
                     action="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_COPY %>">
-                    <%-- Previous Questions --%>
-                    <table class="table-responsive table table-hover table-bordered margin-0" id="copyTableModal">
-                        <thead class="fill-primary">
-                            <tr>
-                                <th style="width:30px;">&nbsp;</th>
-                                <th onclick="toggleSort(this);" id="button_sortid" class="button-sort-ascending"> 
-                                    Course ID <span class="icon-sort sorted-ascending"></span>
-                                </th>
-                                <th onclick="toggleSort(this);" id="button_sortfsname" class="button-sort-none" style="width:17%;">
-                                    Session Name <span class="icon-sort unsorted"></span>
-                                </th>
-                                <th onclick="toggleSort(this);" id="button_sortfqtype" class="button-sort-none"> 
-                                    Question Type <span class="icon-sort unsorted"></span>
-                                </th>
-                                <th onclick="toggleSort(this);" id="button_sortfqtext" class="button-sort-none"> 
-                                    Question Text <span class="icon-sort unsorted"></span>
-                                </th>
-                            </tr>
-                        </thead>
-                        <c:forEach items="${copyQnForm.questionRows}" var="row">
-                            <tr style="cursor:pointer;">
-                                    <td><input type="checkbox"></td>
-                                    <td>${row.courseId}</td>
-                                    <td>${row.fsName}</td>
-                                    <td>${row.qnType}</td>
-                                    <td>${row.qnText}</td>
-                                    <input type="hidden" value="${row.qnId}">
-                            </tr>
-                        </c:forEach>
-                    </table>
-                    <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${copyQnForm.fsName}">
+                    
+                    <input type="hidden" name="<%= Const.ParamsNames.FEEDBACK_SESSION_NAME %>" value="${feedbackSessionName}">
                     <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
-                    <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${copyQnForm.courseId}">
+                    <input type="hidden" name="<%= Const.ParamsNames.COURSE_ID %>" value="${courseId}">
```

Comment:
I removed the line that cleared the form instead. That was a bug, before the first code review, it was used to clear the loading message but I forgot about that.